_pages/volunteer.md

```diff
@@ -0,0 +1,135 @@
+
+- **Learning new skills:** you don’t have to be a Python expert to get involved with pyOpenSci. We can help you level up your packaging game and learn how to constructively review both code and copy through contributions to our online learning resources.
+- **Connecting with peers:** our [public Discourse forum](https://pyopensci.discourse.group/) is a great place to ask and answer questions, as well as discuss packaging and open science with other Pythonistas.
+- **Being recognized** on the [pyOpenSci website](https://www.pyopensci.org/our-community/index.html) and [GitHub repositories](https://github.com/pyOpenSci): your contribution matters, and we want to ensure your work is recognized and celebrated in a public forum.
+
+</div>
+</div>
+
+{% include div_purple_top.html  %}
+
+<div class="pyos-section purple" markdown="1">
+<div class="content" markdown="1">
+
+## Volunteer opportunities with pyOpenSci
+
+There are many ways to get involved with pyOpenSci! We’re always looking for folks to:
+
+{% include feature_row_pyos id="help-us" %}
+
+
+</div>
+</div>
+
+{% include div_purple_bottom.html  %}
+
+
+<div class="pyos-section" markdown="1">
+<div class="content" markdown="1">
+
+## Share your expertise and experience with the broader community through blogging
+
+And last but not least, we’d also love for you to be a guest blogger on the [pyOpenSci blog](https://www.pyopensci.org/blog/index.html)! If you’d like to write about a pyOpenSci package, your experiences with pyOpenSci, or how you’re using free and open Python tools in your scientific endeavors, we’d love to hear from you! Email our Community team at [media@pyopensci.org](media@pyopensci.org) for more information.
+
+## Learn more about volunteering with pyOpenSci
+
+If you still have questions about volunteer roles at pyOpenSci, email our Community team at [media@pyopensci.org](media@pyopensci.org), and we’d be more than happy to help! You can also ask questions and network with the broader pyOpenSci community on our [Discourse forum](https://pyopensci.discourse.group/).
+
+</div>
+</div>
+
+
+<div class="pyos-section purple" markdown="1">
+<div class="content" markdown="1">
+
+## Let's connect!
+
+- [<i class="fa-brands fa-discourse"></i> Discourse](https://pyopensci.discourse.group/)
+- [<i class="fa-brands fa-mastodon"></i> Mastodon](https://fosstodon.org/@pyopensci)
+- [<i class="fa-solid fa-cloud"></i> Bluesky](https://bsky.app/profile/pyopensci.bsky.social)
+- [<i class="fa-brands fa-linkedin"></i> LinkedIn](https://www.linkedin.com/company/pyopensci)
+- [<i class="fa-brands fa-github"></i> GitHub](https://github.com/pyOpenSci)
+
+</div>
+</div>
+
+
+<div class="pyos-section" markdown="1">
+<div class="content" markdown="1">
+
+## Meet the most recent PyOpenSci community contributors
+
+<!-- Get a list of all contribs and sort reverse so newest are first -->
+{% assign new_ppl = site.data.contributors | reverse %}
```

Comment:
This is liquid. rather than the heavier weight JS which will also always include ALL contributors, i love the idea of - meet the newest contribs. that list will always update and it keeps the page content shorter. i also added a little - to date we've had XXX contributors which is also a dynamically generated element!